backend/create_account_table.sql

```diff
@@ -1,6 +1,6 @@
 CREATE TABLE Accounts(
```

Comment:
I would be interested in debating the choice to change id to account_id in the Accounts table. I would argue in favor of keeping the column 'id'. Why the change?